pybricksdev/connections/pybricks.py

```diff
@@ -227,18 +486,18 @@ def _pybricks_service_handler(self, _: int, data: bytes) -> None:
             if self._enable_line_handler:
                 self._handle_line_data(payload)
 
-    async def connect(self, device: BLEDevice):
+    async def connect(self, device):
```

Comment:
Why removing the type hints?

Reply:
This particular place can either be a `BLEDevice` or a `USBDevice`. I could use a `Union` to indicate support for either if you would prefer.

Reply:
yes please